src/SlackGhost.ts

```diff
@@ -115,15 +116,19 @@ export class SlackGhost {
         }
     }
 
-    public async updateFromISlackUser(slackUser: ISlackUser) {
+    public async updateFromISlackUser(slackUser: ISlackUser, config: IConfig) {
         if (!slackUser.profile) {
             return;
         }
         let changed = false;
-        if (slackUser.profile.display_name && this.displayName !== slackUser.profile.display_name) {
-            await this.intent.setDisplayName(slackUser.profile.display_name);
-            this.displayname = slackUser.profile.display_name;
-            changed = true;
+        if (slackUser.profile.display_name) {
+            const newDisplayname = slackUser.profile.display_name + config.display_name_suffix
```

Comment:
Ideally please use template syntax
```suggestion
            const newDisplayname = `${slackUser.profile.display_name}${config.display_name_suffix}`
```

Reply:
If we want to enforce this style, we should change the tslint.json.
Our current config:
```
    "prefer-template": [
      true,
      "allow-single-concat"
    ],
```
It's auto-fixable.
https://eslint.org/docs/rules/prefer-template

Reply:
Yeah, let's do that in another PR!